src/Cms/ContentLock.php

```diff
@@ -40,6 +40,24 @@ public function __construct(ModelWithContent $model)
         $this->data  = $this->kirby()->locks()->get($model);
     }
 
+    /**
+     * Clear lock
+     *
+     * @return bool
+     */
+    public function clear(): bool
```

Comment:
- This should be a protected method as it should never be possible to clear a lock unconditionally from the outside. Only this class may do that.
- The `remove()` and `unlock()` methods should use this method internally to avoid code duplication.
- It may make sense to rename this to `clearLock()` to be more ... clear. 😛

Reply:
Finding the right name for methods kills me :)) 

It was my hesitation when I wrote, thank you, I will resolve it.

Reply:
@lukasbestle Changes done but i cant add unit test with success. I'm getting following error always. Could you help about that please 😇 

> Kirby\Cms\ContentLockTest::testClearLock
> Kirby\Exception\DuplicateException: /test is already locked

Tried following steps on test:

1. impersonate A
2. create lock
3. get current lock
4. remove user A
5. impersonate B
6. get lock to clear
7. create lock
8. get current lock

Reply:
I've simplified the test a bit and now it works. :)

Reply:
You're great, thank you so much 👏 
My unit test looks like spaghetti pasta next to yours 😅

Reply:
Thank you for your help! ❤️